@xen-orchestra/lite/src/stories/web-core/state-hero/vts-error-no-data.story.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties }"
+    :params="[prop('type').required().enum('page', 'card', 'table').preset('card').widget()]"
+  >
+    <VtsErrorHero v-bind="properties" />
+  </ComponentStory>
+</template>
+
+<script lang="ts" setup>
+import ComponentStory from '@/components/component-story/ComponentStory.vue'
+import { prop } from '@/libs/story/story-param'
+import VtsErrorHero from '@core/components/state-hero/VtsErrorNoDataHero.vue'
```

Comment:
Keep the same name as the file name
```suggestion
import VtsErrorNoDataHero from '@core/components/state-hero/VtsErrorNoDataHero.vue'
```